src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/history.html.twig

```diff
@@ -28,11 +28,16 @@
     {% for status in orderForViewing.history.statuses %}
     <tr>
       <td>
-        <span class="badge rounded"
+        <span class="badge rounded d-print-none"
               style="background-color: {{ status.color }}; color: {{ is_color_bright(status.color) ? 'black' : 'white' }};"
         >
           {{ status.name }}
         </span>
+        <span class="badge rounded d-none d-print-inline-block"
+              style="background-color: white; color: black;"
```

Comment:
Should be inside the stylesheet (I already made a @print screen mediaquery some weeks ago)

Reply:
You could add a class if you need to select it easier